Parking/ParkingSpot/doc/spec.md

```diff
@@ -21,33 +21,33 @@ Thus, an entity of type `ParkingSpot` cannot exist without a containing entity o
     + Attribute type: [DateTime](https://schema.org/DateTime)
     + Optional
 
-+ `name` : Name of this parking spot. It can denote the number or label used to identify it within a parking site. 
++ `name` : Name of this parking spot. It can denote the number or label used to identify it within a parking site.
     + Normative References: [https://schema.org/name](https://schema.org/name)
     + Optional
 
-+ `description` : Description about the parking spot. 
++ `description` : Description about the parking spot.
     + Normative References: [https://schema.org/description](https://schema.org/description)
     + Optional
 
 + `location` : Geolocation of the parking spot, represented by a GeoJSON Point.
     + Attribute type: `geo:json`.
     + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
     + Mandatory. Not nullable (if `address` is not defined).  
-    
+
 + `address` : Registered parking spot civic address.
     + Normative References: [https://schema.org/address](https://schema.org/address)
     + Mandatory. Not nullable (if `location` is not defined).
-    
+
 + `status` : Status of the parking spot from the point of view of occupancy.
     + Attribute type: [Text](https://schema.org/Text)
     + Allowed Values: one Of (`occupied`, `free`, `closed`, `unknown`)
     + Metadata:
-        + `timestamp` : Timestamp which reflects the date when the attribute value was obtained. 
+        + `timestamp` : Timestamp which reflects the date when the attribute value was obtained.
               + Type: [DateTime](https://schema.org/DateTime)
               + Optional
-        + `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
+        + `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/master/README.md#TimeInstant)
```

Comment:
https://github.com/telefonicaid/iotagent-node-lib#TimeInstant is shorter and now that we have get totally rid of develop branches is the best way IMHO.